pkg/artifactory/resource/security/resource_artifactory_scoped_token_test.go

```diff
@@ -338,7 +339,7 @@ func TestAccScopedToken_WithTooLongAudiences(t *testing.T) {
 		Steps: []resource.TestStep{
 			{
 				Config:      scopedTokenConfig,
-				ExpectError: regexp.MustCompile(".*Total combined length of audiences field exceeds 255 characters:.*"),
+				ExpectError: regexp.MustCompile(".*total combined length of audiences field exceeds 255 characters:.*"),
```

Comment:
Is the lowercase 'total' intentional? I'm not seeing a case-insensitive regex here. If this test passes, then all's good.

Reply:
Yes it is and yes I fixed the test. Golang spec says that error strings should be lower case